p2p/node/streamManager/streamManager.go

```diff
@@ -188,7 +190,7 @@ func (sm *basicStreamManager) GetHost() host.Host {
 }
 
 // Writes the message to the stream.
-func (sm *basicStreamManager) WriteMessageToStream(peerID p2p.PeerID, stream network.Stream, msg []byte) error {
+func (sm *basicStreamManager) WriteMessageToStream(peerID p2p.PeerID, stream network.Stream, msg []byte, protoversion protocol.ID, reporter libp2pmetrics.Reporter) error {
```

Comment:
we need the metrics for this function as well right? not just the WriteMessageToStream in common module

Reply:
oh yes, good catch

Reply:
fixed